integration/Cargo.toml

```
@@ -0,0 +1,33 @@
+[package]
+name = "k8s-gateway-api-tests"
```

```suggestion
name = "integration"
```

Seems fine for this to have an easier name to reference